two_points_on_the_same_branch.sh

```diff
@@ -373,35 +373,43 @@ fi
 # - for the NEW commit
 # - for the OLD commit
 
+# Generate per line timing info
+export TIMING_ARG=-time-log
+
 # The following variable will be set in the following cycle:
 installable_coq_opam_packages=
 
 for coq_opam_package in $coq_opam_packages; do
     echo "DEBUG: coq_opam_package = $coq_opam_package"
 
-    # perform measurements for the NEW commit (provided by the user)
-    export OPAMROOT="$new_opam_root"
+  for RUNNER in NEW OLD; do
```

Comment:
Indentation seems off here

Reply:
I did not want to re-indent everything (i've added a loop) so used a "half indentation" here...